### PR TITLE
Refactor/before each

### PR DESCRIPTION
**Pull request by irmakerdem**
What does this PR do?
- Completes Hydration and Sleep classes and tests

Where should your reviewer start?
- Line 2 in Hydration.js
- Line 7 in Hydration-test.js
- Line 1 in Sleep.js
- Line 1 in Sleep-test.js

What (if any) features are you implementing?
- Hydration class can:

1. calculate ave ounces consumed per day
2. calculate ounces consumed on a specific day
3. calculate ounces consumed each day over a week

- Sleep class can: 
1. calculate ave sleep hours and quality per user
2. calculate sleep hours and quality for specific day per user
3. calculate hours slept and quality over a week per user
4. calculate ave sleep quality for all users

What (if anything) did you refactor?
- Remove pseudocode
- Fix spacing
- Add global variables inside beforeEach 😭 🤓 ❤️
- change branch names
- clarify method names

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- No

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: test/Sleep-test.js [hunk 1]
```diff
@@ -0,0 +1,358 @@
+import { expect } from 'chai';
```
Comment (catlynbowles) [inline test/Sleep-test.js:1]: very thorough testing, team ! 🦀

File: test/Hydration-test.js [hunk 2]
```diff
@@ -134,83 +136,74 @@ describe('Hydration', () => {
...
     expect(hydration.obtainAveOuncesConsumedPerDay(waterData1)).to.equal(60.14);
 
-    let waterData2 = hydration.obtainHydrationDataBasedOnId(2);
-
     expect(hydration.obtainAveOuncesConsumedPerDay(waterData2)).to.be.a('number');
     expect(hydration.obtainAveOuncesConsumedPerDay(waterData2)).to.equal(39.71);
   });
 
   it('should obtain ounces for most recent day', () => {
-    let waterData1 = hydration.obtainHydrationDataBasedOnId(1);
```
Comment (catlynbowles) [inline test/Hydration-test.js:193]: I love how this was cleaned up with beforeEach. 🐬

**Review by catlynbowles — APPROVED** [review]

File: test/Sleep-test.js (hunk 1 above)
Comment (ASands17) [reply test/Sleep-test.js:1]: Great job getting that beforeEach hook! We learned a lot with that one. 🧠

**Review by ASands17 — APPROVED** [review]
Great work CIA! 😎

**Pull request merged**